src/witan/app/model.clj

```diff
@@ -0,0 +1,77 @@
+(ns witan.app.model
+  (:require [qbits.hayt :as hayt]
+            [qbits.alia.uuid :as uuid]
+            [qbits.alia.codec :as codec]
+            [clj-time.core     :as t]
+            [clj-time.format   :as tf]
+            [clj-time.coerce   :as tc]
+            [witan.app.config :as c]
+            [witan.app.util :as util]
+            [witan.app.schema :as ws]
+            [schema.core :as s])
+  (:use [liberator.core :only [defresource]]))
+
+(def model-table :models)
+(def model-names-table :model_names)
```

Comment:
since we're working with cassandra entities directly in the hayt queries (column names), does it make sense to refactor out the name of the table?